config/config-hd.sh.sample

```diff
@@ -7,63 +7,15 @@ INSTALLER_SBIN_DIR="/inst/installer/sbin"
 
 INSTALL_KERNEL="${INSTALLER_FILES_DIR}/bzImage"
 INSTALL_ROOTFS="${INSTALLER_FILES_DIR}/op3-graphical-builder-genericx86-64.tar.bz2"
-INSTALL_MODULES=""
```

Comment:
We aren't using these, but they are all valid by the installer framework, so they should stay.